spec/syntax.md

```diff
@@ -91,150 +91,198 @@ The syntax specification takes into account the following design restrictions:
 
 _This section is non-normative._
 
-### Messages
+### Messages and Patterns
 
-All messages, including simple ones, are enclosed in `{…}` delimiters:
+A **_message_** is the complete template for a specific message formatting request.
 
-    {Hello, world!}
+All _messages_ MUST contain either a _pattern_ or _selectors_. 
+A _message_ MAY contain zero or more _declarations_.
+(Implication: the empty string is not a valid message)
```

Comment:
This doesn't need to be parenthetical, even though it is re-stating an implication.

```suggestion
As an implication, an empty string is not a valid message.
```